CMakeLists.txt

```diff
@@ -29,6 +29,14 @@ option(RUN_TESTS "Build and run tests" OFF)
 option(ENABLE_CODECOVERAGE "Instrument code for code coverage (only enabled with RUN_TESTS)" OFF)
 option(USE_GETTEXT "Build translation files using gettext" OFF)
 
+if(NINTENDO_SWITCH)

```

Comment:
This should be in the same place where it previously was (just before `if(VITA)`)

Reply:
This was part of my fiddling to try and get this to work identically to the current setup which sets NONET ON [here](https://github.com/diasurgical/devilutionX/blob/master/CMake/switch/switch_defs.cmake#L1) and not externally as I've had to do with this. I guess it's because the current setup includes this file via the toolchain file and my changes mean it's included via CMakeLists.txt?

Reply:
I don't think the current setup includes it via the toolchain file!

Reply:
I think the issue may have been introduced in the commit that added the `DISABLE_TCP` option.

Reply:
Hmm, nevermind that, the `DISABLE_TCP` stuff looks OK.

Anyway, please move this block where it was before and we can investigate the NONET stuff separately after merging this.

Reply:
preferably we would just get libsodium/zt compiling on devkitpro and then not have to disable network :)